src/QuizAtom.tsx

```diff
@@ -1,7 +1,268 @@
-import React from 'react';
+import React, { useState, KeyboardEvent } from 'react';
+import { SvgCheckmark, SvgCross } from '@guardian/src-icons';
+import { css, cx } from 'emotion';
+import { neutral, news } from '@guardian/src-foundations/palette';
+import { body, textSans } from '@guardian/src-foundations/typography';
+import { visuallyHidden } from '@guardian/src-foundations/accessibility';
+import { focusHalo } from '@guardian/src-foundations/accessibility';
 
-import { QuizAtomType } from './types';
+import {
+    QuizAtomType,
+    Question as QuestionType,
+    Answer as AnswerType,
+} from './types';
 
-export const QuizAtom = ({ id }: QuizAtomType): JSX.Element => (
-    <div data-atom-id={id}>QuizAtom</div>
-);
+const neutralBackgroundColour = css`
+    background-color: ${neutral[97]};
+`;
+const darkerNeutralBackgroundColour = css`
+    background-color: ${neutral[86]};
+`;
+const opaqueGreenBackgroundColour = css`
+    background-color: rgb(61, 181, 64);
+`;
+const translucentGreenBackgroundColour = css`
+    background-color: rgb(61, 181, 64, 0.6);
+`;
+const redBackgroundColour = css`
+    background-color: ${news[400]};
+`;
+
+const commonLabelStyle = css`
+    svg {
+        width: 30px;
+        height: 30px;
+    }
+    display: block;
+    padding: 0.75rem 1.25rem;
+    transition: background-colour 100ms;
+`;
+
+const focusLabelStyle = css`
+    cursor: pointer;
+    ${neutralBackgroundColour};
+    :hover {
+        ${darkerNeutralBackgroundColour};
+    }
+    :focus {
+        ${focusHalo}
+        ${darkerNeutralBackgroundColour};
+    }
+`;
+
+const commonTextStyle = css`
+    padding-left: 8px;
+    padding-top: 0;
+    min-height: auto;
+`;
+
+const blackColour = css`
+    color: ${neutral[7]};
+`;
+const whiteColour = css`
+    color: ${neutral[100]};
+`;
+const revealTextStyle = css`
+    ${textSans.xsmall()}
+`;
+
+const fieldsetStyle = css`
+    margin-bottom: 12px;
+    border: 0px;
+    padding: 0px;
+`;
+
+const questionStyle = css`
+    display: flex;
+    div {
+        padding-right: 0.5ch;
+    }
+    legend {
+        margin-bottom: 12px;
+    }
+    span {
+        padding-right: 12px;
+    }
```

Comment:
I personally  would prefer having each style apply to the component directly rather than inherited from a parent component. The issue here is that now we apply the same `padding`/`margin` to all DOM child elements.